src/scales/timeScale.ts

```diff
@@ -47,6 +46,38 @@ export module Scales {
       var startTimeValue = endTimeValue - Plottable.MILLISECONDS_IN_ONE_DAY;
       return [new Date(startTimeValue), new Date(endTimeValue)];
     }
+
+    public scale(value: Date): number {
+      return this._d3Scale(value);
+    }
+
+    protected _getDomain() {
+      return this._d3Scale.domain();
+    }
+
+    protected _setBackingDomain(values: Date[]) {
```

Comment:
`_setDomain()` has the job of making sure that the domain of the scale is set to the according value.  However, this does not necessarily concern setting the domain of whatever backing scale.

I do note that the naming is confusing, and I'm completely open to suggestions on how to change the design of how to implement this idea.